app/controllers/application_controller.rb

```diff
@@ -12,6 +12,11 @@ def disable_nav
     @disable_nav = true
   end
 
+  def queued_jobs
+    @queued_jobs ||= Job.where(approved: false).count
+  end
+  helper_method :queued_jobs
+
```

Comment:
If the queued_jobs is memoized, when does it get reset to include new queued_jobs?